scripts/teamcity-migrate.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -ex
+
+GIT_ID=$(git rev-parse --short=7 HEAD)
+GIT_BRANCH=$(git symbolic-ref --short HEAD)
+REGISTRY=docker.montagu.dide.ic.ac.uk:5000
+NAME=orderlyweb-migrate
+
+TAG=$REGISTRY/$NAME
+COMMIT_TAG=$REGISTRY/$NAME:$GIT_ID
+BRANCH_TAG=$REGISTRY/$NAME:$GIT_BRANCH
+DB=$REGISTRY/montagu-db:$GIT_ID
+
+docker build \
+       --tag $COMMIT_TAG \
+       --tag $BRANCH_TAG \
+       -f migrations/Dockerfile \
+       .
+
+# run this first to avoid a spurious pull error message
+docker push $COMMIT_TAG
+docker push $BRANCH_TAG
+
+# test if we can run the migrations using the registry image - the following is otherwise the same as test-migrate
```

Comment:
It seems slightly backwards that we push the image to our registry before testing whether it works! Not sure what the point of `test-migrate` is either if it's never run on TC. So maybe we just combine them, i.e. add to the top of this script 

```
docker build --tag $TESTING_MIGRATE_IMAGE -f migrations/Dockerfile .

 # Do the migrations
docker run --rm -v ${PWD}/demo:/orderly $TESTING_MIGRATE_IMAGE
```

And then follow with the tagging/building/pushing logic.

Reply:
We took a slightly different approach with the r client that might be of interest: https://github.com/vimc/montagu-r/tree/master/docker

* the `common` file has all the version bits
* there's a build, test and push file that do the steps in order

Reply:
The only point of test-migrate really is for manual local testing. I'll rename it to local-test-migrate.